### PR TITLE
Smoke tests for Data Preservers pallet

### PR DESCRIPTION
**Pull request by nanocryk**
Add smoke tests for this pallet:

- Check deposit match expected values (0 for forced, or the value defined in the runtime config)
- If profile is assigned, the witness variant should match the request variant (Free/StreamPayment)

Feel free to suggest other smoke test ideas.

### CODE REVIEW

**Comment by girazoki** [comment]
@nanocryk data preservers is a pallet common in dancebox, flashbox and dancelight. Therefore we should add the test in the proper folder (smoke-test-common-all)

File: test/suites/smoke-test-common-all/test-data-preservers.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+        it({
+            id: "C02",
+            title: "all assigned profile have assignement witness corresponding to request and whished para id",
+            test: async function () {
+                const entries = await paraApi.query.dataPreservers.profiles.entries();
+
+                for (const [, entry] of entries) {
+                    console.log(JSON.stringify(entry));
+
```
Comment (tmpolaczyk) [inline test/suites/smoke-test-common-all/test-data-preservers.ts:40]:
```suggestion
```

**Comment by tmpolaczyk** [comment]
Test idea: assigned bootnodes have a valid URL (starts with ws or http, not a decode error, etc).

**Comment by girazoki** [comment]
> Test idea: assigned bootnodes have a valid URL (starts with ws or http, not a decode error, etc).

@nanocryk can we add this small test?

**Review by tmpolaczyk — APPROVED** [review]

**Review by girazoki — APPROVED** [review]

**Pull request merged**